Source/Lib/Codec/EbDefinitions.h

```diff
@@ -104,10 +104,10 @@ Groups of Pictures (GOP) units.
 #define EB_PRED_RANDOM_ACCESS   2
 #define EB_PRED_TOTAL_COUNT     3
 #define EB_PRED_INVALID         0xFF
-    
+
 // *Note - This work around is needed for the windows visual studio compiler
-//  (MSVC) because it doesn't support the C99 header file stdint.h.  
-//  All other compilers should support the stdint.h C99 standard types. 
+//  (MSVC) because it doesn't support the C99 header file stdint.h.
+//  All other compilers should support the stdint.h C99 standard types.
 #ifdef _MSC_VER
```

Comment:
one more _MSC_VER?
I recommend do: grep -wr _MSC_VER once again.

Reply:
Some things are msvc only, mingw-w64 is based on gcc